app/src/main/java/com/cornellappdev/transit/models/RouteModels.kt

```diff
@@ -66,4 +68,42 @@ sealed class Transport {
         val secondBus: Int,
         override val lateness: BusLateness
     ) : Transport()
+}
+
+/**
+ * Create a Transport object from Route
+ */
+@RequiresApi(Build.VERSION_CODES.O)
+fun createTransport(route: Route) : Transport {
```

Comment:
To improve autocomplete, you can make this `fun Route.toTransport(): Transport`. That way this would only autocomplete if you correctly had some `route`.

And then you can replace `route` with `this` in the `fun` body